datatypes/src/main/java/org/hyperledger/besu/datatypes/Hash.java

```diff
@@ -50,6 +51,17 @@ public class Hash extends DelegatingBytes32 {
    */
   public static final Hash EMPTY = hash(Bytes.EMPTY);
 
+  /**
+   * Hash of empty requests or "0x6036c41849da9c076ed79654d434017387a88fb833c2856b32e18218b3341c5f"
+   */
+  public static final Hash EMPTY_REQUESTS_HASH =
```

Comment:
Assuming this change for devnet5?

Reply:
Yep this will change for devnet5. Will create another PR for that change though